woocommerce/woocommerce
# Dev: migrate @woocommerce/data reviews to Typescript

### All Submissions:






### Changes proposed in this Pull Request:

Part of #33105

This PR migrates reviews store to TS.

There are no code changes to the code, with some changes to the tests due to impossible values or wrong types of parameters being used in some of the test cases. 

The test case changes were done in a separate commit so that they can be isolated and run separately from the source files type annotation changes.

### How to test the changes in this Pull Request:

1. Should run `pnpm run build && pnpm run test` from `./packages/js/data` without errors
2. Should run `pnpm run ts:check` from `./plugins/woocommerce-admin` without errors

### Other information:




### FOR PR REVIEWER ONLY:
